pytd/spark.py

```diff
@@ -23,8 +23,8 @@ def download_td_spark(spark_binary_version="2.11", version="latest", destination
     destination : str, optional
         Where a downloaded jar file to be stored.
     """
-    td_spark_jar_name = "td-spark-assembly_{}-{}.jar".format(
-        spark_binary_version, version
+    td_spark_jar_name = "td-spark-assembly-{}_spark{}.jar".format(
```

Comment:
Since Python 3.5 is deprecated, how about using f-string?

Reply:
Sounds good. Updated support Python versions accordingly... 45c15524267d73b9ee44431bcd88891452e23668 & 2ff0eae605fef5bd805b654622fae98ba9ab13aa